x/liquidity/spec/03_state_transitions.md

```diff
@@ -1,174 +1,235 @@
+### Stay case
 
-**Stay case**
+Variables:
+
+- `swapPrice` = last `PoolPrice`
+- `EX`: All executable orders that swap X for Y with order price equal to or greater than last `PoolPrice`
+- `EY`: All executable orders that swap Y for X with order price equal or lower than last `PoolPrice`
+
+- **ExactMatch**: If `EX` == `EY`*`swapPrice`
+
+  - Amount of X coins matched from swap orders = `EX`
+  - Amount of Y coins matched from swap orders = `EY`
 
-- Variables
-  - `swapPrice` = last `PoolPrice`
-  - `EX` : All executable orders which swap X for Y with order price equal or higher than last `PoolPrice`
-  - `EY` : All executable orders which swap Y for X with order price equal or lower than last `PoolPrice`
-- **ExactMatch** : If `EX` == `EY`\*`swapPrice`
-  - Amount of X coins matched from swap orders = `EX`
-  - Amount of Y coins matched from swap orders = `EY`
 - **FractionalMatch**
-  - If `EX` > `EY`\*`swapPrice` : Residual X order amount remains
-    - Amount of X coins matched from swap orders = `EY`\*`swapPrice`
-    - Amount of Y coins matched from swap orders = `EY`
-  - If `EY` > `EX`/`swapPrice` : Residual Y order amount remains
-    - Amount of X coins matched from swap orders = `EX`
-    - Amount of Y coins matched from swap orders = `EX`/`swapPrice`
-
-**Increase case**
-
-- Iteration : iterate `orderPrice(i)` of all swap orders from low to high
-
-  - variables
-    - `EX(i)` : Sum of all order amount of swap orders which swap X for Y with order price equal or higher than this `orderPrice(i)`
-    - `EY(i)` : Sum of all order amount of swap orders which swap Y for X with order price equal or lower than this `orderPrice(i)`
-  - ExactMatch : SwapPrice is found between two orderPrices
-    - `swapPrice(i)` = (`X` + 2*`EX(i)`)/(`Y` + 2*`EY(i-1)`)
-      - condition1) `orderPrice(i-1)` < `swapPrice(i)` < `orderPrice(i)`
-    - `PoolY(i)` = (`swapPrice(i)`_`Y` - `X`) / (2_`swapPrice(i)`)
-      - condition2) `PoolY(i)` >= 0
-    - If both above conditions are met, `swapPrice` is the swap price for this iteration
-      - Amount of X coins matched = `EX(i)`
-    - If one of above conditions doesn’t hold, go to FractionalMatch
-  - FractionalMatch : SwapPrice is found at an orderPrice
-    - `swapPrice(i)` = `orderPrice(i)`
-    - `PoolY(i)` = (`swapPrice(i)`_`Y` - `X`) / (2_`swapPrice(i)`)
-    - Amount of X coins matched :
-      - `EX(i)` ← min[ `EX(i)`, (`EY(i)`+`PoolY(i)`)*`swapPrice(i)` ]
-
-- Find optimized swapPrice :
-  - Find `swapPrice(k)` which has the largest amount of X coins matched
+
+  - If `EX` > `EY`*`swapPrice`: Residual X order amount remains
+
+    - Amount of X coins matched from swap orders = `EY`*`swapPrice`
+    - Amount of Y coins matched from swap orders = `EY`
+
+  - If `EY` > `EX`/`swapPrice`: Residual Y order amount remains
+
+    - Amount of X coins matched from swap orders = `EX`
+    - Amount of Y coins matched from swap orders = `EX`/`swapPrice`
+
+### Increase case
```

Comment:
Increase case means... what? when the swap increases the value? or ? the pool is increased? can we describe here what we mean and how we use this?